content/WebAuthn/Supporting_U2F_or_FIDO2_Security_Keys_on_iOS_or_iPadOS/.conf.json

```diff
@@ -0,0 +1,20 @@
+{
+  "order": [
+    "Introduction",
+    "Paths_for_Supporting_U2F_or_FIDO2_Security_Keys",
+    "Security_Key_Compatibility",
+    "Known_Issues",
+    "FAQ"
+
+  ],
+  "suggest-edits": {},
+  "git": [
```

Comment:
what are we trying to acomplish with this "git" config block here?

Reply:
I don't know. I found it in another conf.json file that looked otherwise innocuous. Because the html page didn't seem to have anything unexpected on it, I thought there must be a good reason to include it.